### PR TITLE
fix(tests): fix broken browser tests

### PR DESCRIPTION
**Pull request by rachel-fenichel**
## The basics

- [x] I branched from develop
- [x] My pull request is against develop
- [x] My code follows the [style guide](https://developers.google.com/blockly/guides/modify/web/style-guide)
- [x] I ran `npm run format` and `npm run lint`

### Proposed Changes

- Change offset math to offset by a fixed amount from top-left, rather than a percent
- Use correct input for one of the tests (`DO0` instead of `IF0`)
- Use a different category for one of the drag tests, because one of the blocks in the category was broken. That broken block is also fixed in a different PR/set of PRs.
- Switched from the 'landing' reporter, which the mocha documentation calls a 'gimmick', to one that prints the names of tests to the console as it runs.

#### Behavior Before Change

Tests worked for Beka but not for me

#### Behavior After Change

Hopefully tests work for everyone

### Reason for Changes

### CODE REVIEW

File: tests/browser/test/basic_block_test.js [hunk 1]
```diff
@@ -37,14 +34,8 @@ suite('Basic block tests', function (done) {
...
-      await dragBlockTypeFromFlyout(
-        browser,
-        'Basic',
-        'test_basic_empty',
-        250,
-        50 * i
-      );
-      chai.assert.equal((await getAllBlocks(browser)).length, i);
+      await dragNthBlockFromFlyout(browser, 'Align', 0, 250, 50 * i);
+      chai.assert.equal(i, (await getAllBlocks(browser)).length);
```
Comment (BeksOmega) [inline tests/browser/test/basic_block_test.js:38]: It should be the actual vaue and then the expecte value (swap le arguments).

File: tests/browser/test/basic_block_test.js [hunk 2]
```diff
@@ -37,14 +34,8 @@ suite('Basic block tests', function (done) {
...
     for (let i = 1; i <= 3; i++) {
-      await dragBlockTypeFromFlyout(
-        browser,
-        'Basic',
-        'test_basic_empty',
-        250,
-        50 * i
-      );
-      chai.assert.equal((await getAllBlocks(browser)).length, i);
+      await dragNthBlockFromFlyout(browser, 'Align', 0, 250, 50 * i);
```
Comment (BeksOmega) [inline tests/browser/test/basic_block_test.js:37]: Was there a reason this needed to be switched out?

File: tests/browser/test/basic_playground_test.js [hunk 3]
```diff
@@ -182,7 +182,7 @@ suite('Disabling', function () {
         110,
         110
       );
-      await connect(browser, child, 'PREVIOUS', parent, 'IF0');
```
Comment (BeksOmega) [inline tests/browser/test/basic_playground_test.js:185]: Huh, I guess this was just getting it close enough that it was working? lol

**Review by BeksOmega — APPROVED** [review]
Works on my machine too!

File: tests/browser/test/basic_block_test.js (hunk 2 above)
Comment (rachel-fenichel) [reply tests/browser/test/basic_block_test.js:37]: One of the basic blocks was breaking (I think it's one that you fixed in dev-tools) and it was making testing difficult.

I want to have a different test to validate that every test block can be shown in the toolbox and dragged in, but this particular test doesn't actually care which block you use.

File: tests/browser/test/basic_block_test.js (hunk 1 above)
Comment (rachel-fenichel) [reply tests/browser/test/basic_block_test.js]: Done. My expectations were backwards!

File: tests/browser/test/basic_playground_test.js (hunk 3 above)
Comment (rachel-fenichel) [reply tests/browser/test/basic_playground_test.js:185]: Yup.

**Pull request merged**